src/components/BurgerMenu/BurgerMenu.css

```diff
@@ -0,0 +1,21 @@
+@import '../palette.css';
+
+.drawer {
+  & > div {
+    &:last-of-type {
```

Comment:
this could be 
```css
.drawer > div:last-of-type {
  rule;
}
```
if you're only going to override material ui style. Also, please notate that you are overriding material ui style here for posterity.